pyproject.toml

```diff
@@ -25,11 +25,15 @@ classifiers = [
 python = "^3.7"
 zeep = "^4.1.0"
 requests = "^2.27.1"
+flake8 = {version = "^4.0.1", optional = true}
```

Comment:
Can you move this to dev-dependencies rather than primary library dependency?

Reply:
Sorry, should have mentioned that, for Poetry extras to work, the optional dependency cannot be a dev-only dependency. See the following: https://stackoverflow.com/a/66030875/7666672

In hindsight, it's a lot simpler to keep it as a regular dev dependency.

Reply:
All good.  We have working CI for automated tests now.  Thanks for the PR!   With our merged updates first CI has finished successfully.  🍾